### PR TITLE
CLI Import - workaround for BZ #1263650 (wait for CV publish)

### PR DESCRIPTION
**Pull request by lpramuk**
CLI Import - workaround for BZ #1263650 (wait for CV publish)

+ added wait_for_publish() method: waits for all Publish tasks
* import_content_hosts(): incorporated bug workaround

(Issue #2943)

### CODE REVIEW

**Comment by lpramuk** [comment]
```
$ nosetests -vs tests.foreman.cli.test_import:TestImport.{test_import_content_hosts_default,test_reimport_content_hosts_negative}
...
----------------------------------------------------------------------
Ran 2 tests in 639.523s

OK
```

**Comment by sthirugn** [comment]
ACK

File: tests/foreman/cli/test_import.py [hunk 1]
```diff
@@ -218,6 +220,16 @@ def get_sat6_id(
     return entity_dict
 
 
+def wait_for_publish(delay_step=30):
```
Comment (sthirugn) [inline tests/foreman/cli/test_import.py]: Non blocker comment: This may be usesful for other cli tests too, may be we can move this to helpers.py

**Comment by rplevka** [comment]
ACK

**Pull request merged**